libs/datamodel/core/src/common/constraint_names.rs

```diff
@@ -67,13 +66,13 @@ impl ConstraintNames {
             .collect();
 
         idx.db_name.as_ref().unwrap()
-            == &ConstraintNames::index_name(model.final_database_name(), &column_names, idx.tpe, connector)
+            == &ConstraintNames::index_name(model.final_database_name(), &column_names, idx.is_unique(), connector)
     }
 
     pub(crate) fn index_name(
         table_name: &str,
         column_names: &[&str],
-        tpe: IndexType,
+        is_unique: bool,
```

Comment:
Boolean parameters... I didn't like the `IndexType` either. Maybe a separate function for unique index?

Reply:
Yep that was the plan, I forgot. Will make it happen before merging.